src/Microsoft.ML.Data/Evaluators/Metrics/BinaryClassificationMetrics.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// Evaluation results for binary classifiers, excluding probabilistic metrics.
+    /// Evaluation results for binary classifiers (supervised learning algorithm), excluding probabilistic metrics.
```

Comment:
>(supervised learning algorithm), [](start = 50, length = 32)

I'd remove this. You can make an unsupervised binary classifier too, technically, and these metrics would work well for those too.